src/api/.jshintignore

```diff
@@ -2,3 +2,4 @@ app/assets/javascripts/webui/application/bento
 app/assets/javascripts/webui/application/cm2
 app/assets/javascripts/webui2/cm2
 app/assets/javascripts/webui2/*.min.js
+app/assets/javascripts/webui2/monitor.js
```

Comment:
why?

Reply:
I just wanted to move the JS code from the view to its own assets file. I addressed some of the lint errors, however there are some more to fix and our goal is only migrating the view and not rewriting the code.

Reply:
then file an issue with the rest and take it as a comment here? Just adding it to the ignore file sounds dangerous

Reply:
It was ignored before, it is ignored now. Doesn't sound very dangerous to me :)

Reply:
Well, all of webui was ignored - not so for webui2